staging/src/k8s.io/apimachinery/pkg/util/validation/field/path.go

```diff
@@ -16,76 +16,12 @@ limitations under the License.
 
 package field
 
-import (
-	"bytes"
-	"fmt"
-	"strconv"
-)
+import "k8s.io/utils/field"
 
 // Path represents the path from some root to a particular field.
-type Path struct {
-	name   string // the name of this field or "" if this is an index
-	index  string // if name == "", this is a subscript (index or map key) of the previous element
-	parent *Path  // nil if this is the root element
-}
+type Path = field.Path
 
 // NewPath creates a root Path object.
 func NewPath(name string, moreNames ...string) *Path {
```

Comment:
Why leave this shim in place?  I know there are a lot of callers of this, but it seems like something gorename (or sed) could fix?

Reply:
This affects bascially all 3rdparty API developers. You see it in our code with a hundred uses or so.